packages/bridge-status-controller/CHANGELOG.md

```diff
@@ -7,9 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Added
+## [12.0.1]
+
+### Fixed
 
-- Add `relay` to the list of bridges in the `BridgeId` enum ([#5636](https://github.com/MetaMask/core/pull/5623))
+- Add `relay` to the list of bridges in the `BridgeId` enum to prevent validation from failing ([#5636](https://github.com/MetaMask/core/pull/5636))
```

Comment:
```suggestion
- Add `relay` to the list of bridges in the `BridgeId` enum to prevent validation from failing ([#5623](https://github.com/MetaMask/core/pull/5623))